src/Context/Context.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { createContext, useEffect, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+
+interface PeopleContextType {
+  people: Person[];
+  loader: boolean;
+  loadingError: boolean;
+  setPeople: (people: Person[]) => void;
+}
+
+export const PeopleContext = createContext<PeopleContextType>({
+  people: [],
+  loader: false,
+  loadingError: false,
+  setPeople: () => {},
```

Comment:
The `setPeople` function is initialized as an empty function in the `PeopleContext`. This might lead to unexpected behavior if it's called before being properly set. Consider initializing it with a function that matches the expected type signature or ensure it's set correctly before use.